docs/blog/design-philosophy/index.qmd

```diff
@@ -0,0 +1,245 @@
+
+The table **Heading** is all important to us, and far too many software applications that allow for summary table creation make it difficult or impossible to include a heading. The **Footer** is equally important but is unfortunately often neglected. It provides a place for table notes and will only be present in **Great Tables** if table notes do exist.
+
+The **Spanner Labels** are pieces of text that span across **Column Labels**, allowing for some grouping of columns where it makes sense. This is important for the readability of tables, especially wider tables having many columns.
+
+Aside from a table's stuctural considerations, we also believe that the data values contained within the cells often require formatting before presentation, since the representation for computing isn't always optimal for readers of a table. To that end, we provide a large collection of formatting methods that are purpose-built for each type of formatting operation. While focused on simplicity they are also powerful, giving you the ability to customize values to your liking. We want users all over the world to be able to use **Great Tables** successfully so we built in locale-based formatting options, based on the latest guidance from the [Unicode CLDR](https://cldr.unicode.org).
+
+The majority of open-source software packages that deal with tables seem to be more focused on the display of simple yet large tables. We wanted to be different in this regard and instead focus on the task of generating summary tables. Those open-source table projects typically produce tables with pagination and other such controls to navigate and filter hundreds, thousands, maybe even more records. And that is great for those situations where you'd want to expose a database of records for a user to explore. But we wanted to go the other way and help the type of user that wanted to present smaller, summarized data. This is typically what you see in journal articles, in books, and in periodicals like [*The Economist*](https://www.economist.com/economic-and-financial-indicators/2024/03/14/economic-data-commodities-and-markets). We long held that there was an unaddressed need for users to create highly customized tables that could serve as interesting visualizations.
+
```

Comment:
> [*The Economist*](https://www.economist.com/economic-and-financial-indicators/2024/03/14/economic-data-commodities-and-markets)

If you make the link https://www.economist.com/economic-and-financial-indicators/, it'll link to a list of the most recently market and financial indicators, not just last week's.